src/index.js

```diff
@@ -1,2 +1,2 @@
-export Notification from './notification';
-export NotificationStack from './notificationStack';
+export { Notification } from './notification';
+export { NotificationStack } from './notificationStack';
```

Comment:
Are these semantics correct? Note that both Notification and NotificationStack are default exports and not named exports.

I believe the correct syntax is:

```js
export { default as Notification } from './notification';
```

https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=true&presets=es2015%2Creact&code=export%20%7B%20default%20as%20a%20%7D%20from%20'.%2Fa'%3B%0Aexport%20%7B%20a%20%7D%20from%20'.%2Fa'%3B